src/flint/fq_default_mpoly.jl

```diff
@@ -22,6 +24,34 @@ mutable struct fq_default_mpoly <: MPolyElem{fq_default}
     end
 end
 
+macro fq_default_mpoly_do_op(f, R, a)
```

Comment:
With some `...` magic this could be squashed into one macro I think. But it is not too important.